core/src/main/java/org/arquillian/cube/impl/client/container/CubeContainerLifecycleController.java

```diff
@@ -24,7 +24,8 @@
     @Inject
     private Event<CubeControlEvent> controlEvent;
 
-    public void startCubeMappedContainer(@Observes BeforeStart event, CubeRegistry cubeRegistry,
+    // Must be started before ContainerConfigurationController
+    public void startCubeMappedContainer(@Observes(precedence=100) BeforeSetup event, CubeRegistry cubeRegistry,
```

Comment:
Ok.  At the moment, I'm letting the socket pick a random port, so I'd need to figure out a way to allocate a port ahead of time....something to figure out.